csharp/Svix/Endpoint.cs

```diff
@@ -252,14 +252,15 @@ public string GetSecret(string appId, string endpointId, string idempotencyKey =
             }
         }
 
-        public List<EndpointOut> List(string appId, ListOptions options = null, string idempotencyKey = default)
+        public List<EndpointOut> List(string appId, Ordering order = default, ListOptions options = null, string idempotencyKey = default)
```

Comment:
1. Doesn't this break things if you put them before other params?
2. Also, it should be part of ListOptions, not a different param.

Reply:
Updated, now ordering is part of `ListOptions` in all libs. Ruby does not appear in the list because it's all implicitly done in it.